### PR TITLE
Follow-up edits to PR#2475

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/2475

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@tnguyen-rh @bfallonf @tpoitras peer review please

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -795,13 +795,15 @@ user access to a particular project.
 ifdef::openshift-enterprise[]
 [NOTE]
 ====
-With Aggregated Logging version 3.2.1, Curator is available for use as Tech Preview.
-To start it, after completing an installation using the 3.2.1 Deployer, scale up
-the Curator deployment config that was created (defaults to zero replicas).
+With Aggregated Logging version 3.2.1, Curator is available for use as Tech
+Preview. To start it, after completing an installation using the 3.2.1 Deployer,
+scale up the Curator deployment configuration that was created. (It defaults to
+zero replicas).
```
Comment (tnguyen-rh) [inline install_config/aggregate_logging.adoc]: Should the full stop be inside the parens?

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -810,11 +812,11 @@ $ oc scale dc/logging-curator-ops --replicas=1
...
-Curator allows administrators to configure scheduled Elasticsearch maintenance operations
-to be performed automatically on a per-project basis. It is scheduled to perform
-actions daily based on its configuration. Only one Curator pod is recommended per
-Elasticsearch cluster. Curator is configured via a mounted yaml configuration
-file in the following structure:
+Curator allows administrators to configure scheduled Elasticsearch maintenance
+operations to be performed automatically on a per-project basis. It is scheduled
+to perform actions daily based on its configuration. Only one Curator pod is
+recommended per Elasticsearch cluster. Curator is configured via a mounted YAML
+configuration file in the following structure:
```
Comment (tnguyen-rh) [inline install_config/aggregate_logging.adoc]: maybe `s/in/with/`

**Comment by tnguyen-rh** [comment]
@ahardin-rh Couple of nits, otherwise LGTM. :+1:

File: install_config/aggregate_logging.adoc (hunk 2 above)
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: Good catch!

File: install_config/aggregate_logging.adoc (hunk 1 above)
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: Good eye!

**Comment by ahardin-rh** [comment]
Revision history in original PR

**Pull request merged**